openmc/arithmetic.py

```diff
@@ -644,10 +656,19 @@ def nuclides(self):
     def aggregate_op(self):
         return self._aggregate_op
 
+    @property
+    def name(self):
+
+        # Append each nuclide in the aggregate to the string
+        names = [nuclide.name if isinstance(nuclide, Nuclide) else str(nuclide)
+                 for nuclide in self.nuclides]
+        string = '(' + ', '.join(map(str, names)) + ')'
```

Comment:
`map(str, names)` seems unnecessary since the list comprehension above looks like it only contains strings anyway